src/js/user-profile/components/TabItem.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { IndexLink } from 'react-router';
+
+class TabItem extends React.Component {
```

Comment:
In the past, if two components were closely tied together, for example if a `TabItem` is exclusively used with a `TabNav` and never used independently, I've simply defined the child (`TabItem`) in the same file without exporting it. This makes for less components in the folders and saves having to jump around to the child components unnecessarily.

Not sure what you think about this approach; just some food for thought.